db/migrate/20200805102249_validate_add_missing_foreign_keys_to_ahoy_models.rb

```diff
@@ -0,0 +1,8 @@
+class ValidateAddMissingForeignKeysToAhoyModels < ActiveRecord::Migration[6.0]
+  def change
+    validate_foreign_key :ahoy_events, :ahoy_visits
+    validate_foreign_key :ahoy_events, :users
```

Comment:
Since we have invalid data in the database, won't this fail? I think we may need to separate the validating into a separate PR.

Reply:
Maybe we could create a PR that cleans the DB and then merge this one. What do you think?

Reply:
That sounds good to me!

Reply:
This is good now